### ISSUE TITLE
Error running `pop up parachain -f ./network.toml`:  error decoding response body: invalid type: map

### ISSUE THREAD

**Issue by chungquantin** [r0gue-io/pop-cli#138]
My project is created using the command `pop new parachain my-app`, after running the command to deploy the local relaychain `pop up parachain -f ./network.toml`, I got the below error:
```
┌   Pop CLI : Deploy a parachain
│
Error: error decoding response body: invalid type: map, expected a sequence at line 1 column 0

Caused by:
    invalid type: map, expected a sequence at line 1 column 0
```

**Comment by AlexD10S** [comment]
Hi! First of all thanks for reporting issues and helping us to improve pop-cli.

I am trying to replicate your issue, but with the flow you describe is working for me.

- I assume your `network.toml` is the one generated by pop-cli, but can you share your `network.toml`? so I can see if there is something wrong in the generation of it.

- Also I'd appreciate more information with about system and in what commit or branch you installed `pop-cli`

**Comment by chungquantin** [comment]
@AlexD10S Thank you for our response. Below is my `network.toml`.

<img width="568" alt="Screenshot 2024-05-04 at 20 14 43" src="https://github.com/r0gue-io/pop-cli/assets/56880684/7e99fabd-6ca5-42d5-8db4-b3a75b24c2a1">

Actually, I added a few other pallets to my runtime and it builds successfully. I expect the error with `pop up parachain` won't do anything with the runtime. 

My pop cli version is `pop-cli 0.1.0`

**Comment by AlexD10S** [comment]
The `network.toml` is ok, so if you added some pallets and builds successfully `pop up parachain` should work fine.
I have been researching the error `invalid type: map, expected a sequence at line 1 column 0` and it looks it might be an error when parsing some external data ([StackOverflow answer](https://stackoverflow.com/questions/48595735/invalid-type-map-expected-a-sequence-when-deserializing-a-nested-json-struct )).

I have started a new branch in the repo to try to replicate your issue and fix the problem. Can you test the app using the branch `fix/error-parsing`?
You can install it with:
```
cargo install --locked --git https://github.com/r0gue-io/pop-cli --branch fix/error-parsing --force
```
It might be an issue when fetching the Polkadot releases from Github:
https://api.github.com/repos/paritytech/polkadot-sdk/releases

**Comment by chungquantin** [comment]
@AlexD10S Thank you. It is fixed!

**Comment by AlexD10S** [comment]
It seems that you can't parse the response from https://api.github.com/repos/paritytech/polkadot-sdk/releases

It shouldn't be the changes you made in the runtime, but just in case, what changes you made?

**Comment by chungquantin** [comment]
@AlexD10S I only made changes to the consensus and node service. So as I understand, POP uses Github API to install or fetch  the Polkadot SDK and my IP is blocked by Github due to rate limit. 
<img width="1330" alt="Screenshot 2024-05-07 at 14 24 31" src="https://github.com/r0gue-io/pop-cli/assets/56880684/fb24f466-1671-4b4c-8eea-0da4362eb15c">

**Comment by AlexD10S** [comment]
Ok, I see now this is the reason. Will fix it

**Comment by AlexD10S** [comment]
Should be fixed now, you can install it again with
` ` ` 
cargo install --locked --git https://github.com/r0gue-io/pop-cli --branch fix/error-parsing --force
` ` `

**Comment by AlexD10S** [comment]
@chungquantin Thank you once again for your testing efforts. Your contributions are invaluable in helping us continue to enhance the tool. 
I have submitted a fix for this issue: https://github.com/r0gue-io/pop-cli/pull/142
Could you confirm that the entire process is functioning correctly? I emulate the Github API response is a wrong and works, but not sure how to exactly replicate your issue to test the whole process.

**Comment by chungquantin** [comment]
@AlexD10S Hi, after installing the fixed version, I still get the same error. 
```
Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("github.com")), port: None, path: "/paritytech/polkadot-sdk", query: None, fragment: None }
https://api.github.com/repos/paritytech/polkadot-sdk/releases
Response
Error: Anyhow error: error decoding response body: invalid type: map, expected a sequence at line 1 column 0

Caused by:
    0: error decoding response body: invalid type: map, expected a sequence at line 1 column 0
    1: invalid type: map, expected a sequence at line 1 column 0
```

**Comment by AlexD10S** [comment]
Something wrong with the installation there, the logs should not appear. Take the installation from the main branch: 
```
cargo install --locked --git https://github.com/r0gue-io/pop-cli
```

**Comment by ltfschoen** [comment]
i got the same error.
i entered a docker container containing the pop cli binary in folder /pop,
then ran `./target/release/pop new parachain`
and it successfully created a parachain in a subfolder ./myparachain
then i deleted that subfolder
and then tried to create another parachain but it gave that error
```
root@pop-cli:/pop# ./target/release/pop new parachain
T   Pop CLI : Generate a parachain
|
o  Select a template provider:
|  Pop
|
o  Select the type of parachain:
|  Standard
|
Error: error decoding response body

Caused by:
    invalid type: map, expected a sequence at line 1 column 0
```
